unified-codes/apps/web/src/components/molecules/Footer.tsx

```diff
@@ -1,25 +1,37 @@
 import * as React from 'react';
 import { Link as RouterLink } from 'react-router-dom';
 import Link from '@material-ui/core/Link';
-import { Theme, withStyles } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/core/styles';
 import { ClassNameMap } from '@material-ui/core/styles/withStyles';
 
-import { Drawer, Grid, TMFLogo } from '@unified-codes/ui';
+import { Grid, Paper, TMFLogo } from '@unified-codes/ui';
+import { flexDirection, ITheme, position } from '../../muiTheme';
 
 export interface FooterProps {
   classes: ClassNameMap<any>;
 }
 
-type flexDirection = 'column' | 'row';
-
-const getStyles = (theme: Theme) => ({
-  root: {},
+const getStyles = (theme: ITheme) => ({
+  root: {
+    height: 120,
+    position: 'fixed' as position,
+    width: '100%',
+    bottom: 0,
+    alignItems: 'center',
+    backgroundColor: theme.palette.background.footer,
+    display: 'flex',
+    flexDirection: 'column' as flexDirection,
```

Comment:
I was initially hoping to try to avoid components having baked in `flex` styles and only use the material UI grid API: https://material-ui.com/api/grid/. My thinking is it could get messy if we mix `Grid` flexbox with `display: 'flex'` etc. Is there something we can't do here using the grid layout system, or just what was familiar?